### PR TITLE
Remove VS 140, 150 Projects

### PR DESCRIPTION
**Pull request by teksturi**
Remove old Visual Studio project files for 140 and 150. This is done because
Poco will move on to C++17 and older Visual Studio does not support it.
It is still possible to generate these files.

Fixes: #4274 

---

If something else should also be deleted like root/build_vs140.cmd just say
and I will delete those as well. Also review in commit level. Will be easier
as they are logically grouped and do only one thing per commit.

### CODE REVIEW

File: ProGen/rebuildall32.bat [hunk 1]
```diff
@@ -9,11 +9,8 @@ mkdir ProGen\log
 
 call ProGen\progen.bat Win32 >ProGen\log\progen32.log 2>&1
 rd /q /s lib bin     & call buildwin  90 rebuild all both Win32 samples tests devenv  > ProGen\log\90.x32.log 2>&1
```
Comment (matejk) [inline ProGen/rebuildall32.bat:11]: Should the "90"(Visual Studio 2008) project files be removed also? Same applies to the .gitignore.

Comment (teksturi) [reply ProGen/rebuildall32.bat:11]: I do not know. See these for more info.

https://github.com/pocoproject/poco/pull/4342#issuecomment-1854714427
https://github.com/pocoproject/poco/pull/4323#pullrequestreview-1766477057

Comment (matejk) [reply ProGen/rebuildall32.bat:11]: I see. @aleks-f would know.

Comment (aleks-f) [reply ProGen/rebuildall32.bat:11]: The only purpose for the VS90 projects is use  as a template by ProGen - it launches the VS conversion tool and gives it VS90 project file as a template to generate a later VS project versions (with some "massaging" in between). Nothing is built with VS90 projects anymore.

**Pull request merged**